.github/workflows/github-actions-demo.yml

```diff
@@ -0,0 +1,20 @@
+name: GitHub Actions Demo
```

Comment:
It's wrong 

Where is the build step running 

Npm install 

And test step running 
Npm run test?